.github/workflows/github.yml

```diff
@@ -151,6 +151,35 @@ jobs:
       if: ${{ startsWith(matrix.platform, 'msvc') }}
       run: source '${{github.workspace}}/CI/install_vcpkg_dependencies.sh' '${{matrix.platform}}'
 
+    - name: Setup Ninja (Windows)
+      if: ${{ startsWith(matrix.platform, 'msvc') }}
+      uses: seanmiddleditch/gha-setup-ninja@v5
+
+    - name: Setup MSVC Developer Command Prompt
+      if: ${{ startsWith(matrix.platform, 'msvc') }}
+      uses: ilammy/msvc-dev-cmd@v1
+      with:
+        arch: ${{ matrix.arch }}
+
+    - name: Setup sccache (Windows/MSVC)
+      if: ${{ startsWith(matrix.platform, 'msvc') }}
+      uses: hendrikmuhs/ccache-action@v1.2
+      with:
+        variant: sccache
+        key: sccache-${{ matrix.platform }}-${{ github.ref_name }}
+        restore-keys: |
+          sccache-${{ matrix.platform }}-${{ github.ref_name }}
```

Comment:
Perhaps use same naming scheme as non-sccache step? One that wont cause conflicts when we have both develop and beta branches?